src/Badge/Badge.tsx

```diff
@@ -27,15 +31,12 @@ export type BadgeProps = {
   /** Text value to be displayed by badge, defaults to empty. */
   value?: React.ReactNode;
 
-  /** Function called when pressed on the badge. */
-  onPress?: (...args: any[]) => any;
-
```

Comment:
Pls updated the props over here as well.

Reply:
`InlinePressableProps` this includes onPressIn and onPressOut